Dockerfile

```diff
@@ -580,6 +581,10 @@ RUN \
   --disable-debug \
   --disable-shared \
   --disable-ffplay \
+  --disable-manpages \
+  --disable-podpages \
+  --disable-large-tests \
+  --enable-small \
```

Comment:
Oh got much smaller than i would expect. Any idea how much this affect speed?

Reply:
i didn't test, you mean operating speed, conversion speed?

Reply:
Yes if it affects transcoding speed, i think most users prefer speed over size. For codecs built separetly (not built with `-Os`) i guess it should not affects but not sure about native ffmpeg codecs.

Reply:
Still a bit confused how it can be 6MB difference, would be interesting to know what differs so much, can it really be that much smaller in opcodes or are there some big static tables that are done at runtime instead?